docker/api/daemon.py

```diff
@@ -123,9 +123,9 @@ def login(self, username, password=None, email=None, registry=None,
         # If dockercfg_path is passed check to see if the config file exists,
         # if so load that config.
         if dockercfg_path and os.path.exists(dockercfg_path):
-            self._auth_configs = auth.load_config(dockercfg_path)
+            self._auth_configs = self.reload_config(dockercfg_path)
```

Comment:
Oh, duh. I was thinking of code re-use and obviously did not test this part of the code. Good catch. Since I'm moving this to the APIClient I'll just revert these two changes in the login func.